### ISSUE TITLE
Page title has an extra [FSI] Unicode character "Screenshot: [FSI]..." in Windows taskbar name

### ISSUE THREAD

**Issue by cpeterso** [mozilla-services/screenshots#4863]
When viewing a Screenshot page (such as https://screenshots.firefox.com/t286AJZDpqHVxol2/localhost) on Windows 10 (in Firefox, Chrome, or Edge), the Windows 10 taskbar shows the page title with an extra `FSI` Unicode character (U+2068 FIRST STRONG ISOLATE?) like "Screenshot: [FSI]..."

![screenshot_fsi](https://user-images.githubusercontent.com/629193/45320986-efa11300-b4f8-11e8-848c-55a9f57481ef.png)

**Comment by clouserw** [comment]
Can you replicate this on a page we can access to test it?

**Comment by cpeterso** [comment]
> Can you replicate this on a page we can access to test it?

Do you have access to https://screenshots.firefox.com/t286AJZDpqHVxol2/localhost ? I can access that screenshot page in Private Browsing mode, so I assume anyone should be able to see it.

I see this issue in every screenshot page I've tested in Firefox, Chrome, and Edge.

**Comment by chenba** [comment]
The bug is only on the shot page, i.e. My Shots and the home page do not have this bug.  Additionally, the page title ends with a `PDI`.

**Comment by clouserw** [comment]
This is similar to https://github.com/mozilla/testpilot/issues/875

**Comment by clouserw** [comment]
@stasm have you seen this before?

**Comment by stasm** [comment]
Yeah, it's https://github.com/projectfluent/fluent.js/issues/173. Windows doesn't make these control characters invisible. The work-around is to pass `useIsolating: false` to the `MessageContext` constructor:

```js
let ctx = new MessageContext(locale, {useIsolating: false});
```

Ideally, you'd only do this for users on Windows, so that users on other platforms can benefit from bidi isolation of content interpolated in translations.

_Sidenote: In `fluent` 0.8, `MessageContext` was renamed to `FluentBundle`. I'll open an upgrade PR soon._

**Comment by cpeterso** [comment]
How long until the useIsolating fix will be live? I can still reproduce this bug (e.g. when viewing a screenshot page such as https://screenshots.firefox.com/HsHK7fiovYaDBPtf/play.google.com).
